README.md

```diff
@@ -52,8 +54,16 @@ deobfuscates a dalvik executable
  -v,--verbose <LEVEL>            Set verbosity to LEVEL, default: 0
 ```
 
+NOTES: 
+• simplify.jar must be built first. See the next section for more informations.
+
+• Name of the jar may vary. After the build check the simplify/build/libs folder, copy its path and the jar name and paste the entiere path (under quotes) as executable instead of simplify.jar if java cannot locate it itself.
```

Comment:
This is really just explaining how gradle / build systems work. I think it's redundant.

Reply:
Personally I join you for the first point. It was really obvious. But should I really suppress the second also? This may be necessary for more noob users...

Reply:
You could take this info and add it into the quote below:
> The Simplify jar will be in `simplify/build/libs/simplify.jar`. You can test it's working by simplifying the obfuscated-example app:

To something more like:
> The Simplify jar will be in `simplify/build/libs/`. You can test it's working by simplifying the obfuscated-example app. Here's how you'd run it (you may need to change `simplify.jar`):

What do you think?

Reply:
Good idea, I didn't think about that because I thought it was better not to rename the obtained simplify.jar. But if you propose it, this is because it isn't harmful. So I will put that instead. Thanks!